finalPrep/modZeroHero.js

```diff
@@ -74,11 +118,35 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 
 // - Create 2 instances of your SuperHero class
 
-
-// Reflection
-// What parts were most difficult about this exerise?
-
-// What parts felt most comfortable to you?
-
-// What skills do you need to continue to practice before starting Mod 1?
-
+class SuperHero {
+  constructor(name, superpower, age) {
+    this.name = name
+    this.superpower = superpower,
+    this.age = age
+  }
+  static archNemesis = "Syntax error"
+  static powerLevel = 100
+  static energyLevel = 50
+
+  sayName () {
+    return this.name
+  }
+
+  static maximizeEnergy () {
+    this.energyLevel = 1000
+    console.log(this.energyLevel)
+  }
+
+  static gainPower (num) {
+    var t = this.powerLevel
```

Comment:
This method currently updates a variable's value called `t`. However, we want to update the value of `this.power`. You are close- just do to `this.power` what you are currently doing to `t`.

Reply:
I'll look back at the code-that may have been a typo. Thank you!